docs/how-to-guides/upgrade-openebs.md

```diff
@@ -0,0 +1,248 @@
+# Upgrading and migrating OpenEBS data plane components
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Step 1: Upgrade OpenEBS control plane components](#step-1-upgrade-openebs-control-plane-components)
+* [Step 2: Upgrade OpenEBS data plane components](#step-2-upgrade-openebs-data-plane-components)
+	* [Upgrade cStor pools](#upgrade-cstor-pools)
+	* [Upgrade cStor volumes](#upgrade-cstor-volumes)
+* [Step 3: Verify](#step-3-verify)
+* [Summary](#summary)
+* [Additional resources](#additional-resources)
+
+## Introduction
+
+This guide provides steps to upgrade OpenEBS control plane and data plane
+components from one version to another.
+
+In this document we are going to update from `v1.12.0` to `v2.2.0`.
+
+## Prerequisites
+
+To upgrade the OpenEBS data plane components, we need the following:
+
+* A Kubernetes cluster accessible via `kubectl`.
+
+* [OpenEBS](../configuration-reference/components/openebs.md) installed. You can
+  check if OpenEBS is indeed in expected version:
+    ```bash
+    $ kubectl get pods -n openebs -l openebs.io/version=1.12.0
+    ```
+
+* Highly recommended to schedule a downtime for the applications consuming the
+  OpenEBS PV and make sure to take a backup of the data before starting the
+  below upgrade procedure. Lokomotive provides
```

Comment:
Is the disruption expected from the upgrade process? Or may only occur if some problems arise?

Reply:
Disruption is not expected, only if there is a problem. A precautionary measure.

Reply:
I'd mention it somehow then. Maybe:
```suggestion
* Upgrade process should not be disruptive, but it is recommended to schedule a downtime for the applications 
  consuming the OpenEBS PV and take a backup of the data before starting the
  below upgrade procedure, in case some problem arises.
  
  Lokomotive provides
```